software/clipbucket.yml

```
@@ -0,0 +1,12 @@
+name: "ClipBucket"
+website_url: "https://clipbucket.oxygenz.fr/en/"
+source_code_url: "https://github.com/MacWarrior/clipbucket-v5"
+description: "Start your own Video Sharing website (YouTube/Netflix Clone) in a matter of minutes."
+licenses:
+  - AAL
+platforms:
+  - Docker
```

```suggestion
platforms:
  - Docker
  - PHP
```

Manual install instructions also available

> https://github.com/MacWarrior/clipbucket-v5?tab=readme-ov-file#on-dedicated-server

Also disregard errored checks caused by `ERROR:awesome_lint.py: ass: the project is archived`, unrelated to this PR